### PR TITLE
Updated project dependencies

### PR DESCRIPTION
**Pull request by dekelev**

### CODE REVIEW

**Comment by dekelev** [comment]
Hi @daffl, Is it possible to disable the requirement for at least 1 reviewer?

File: package.json [hunk 1]
```diff
@@ -1,7 +1,7 @@
 {
   "name": "feathers-cassandra",
   "description": "Feathers service adapter for Cassandra DB based on Express-Cassandra ORM and CassanKnex query builder",
-  "version": "3.4.2",
+  "version": "3.4.3",
```
Comment (daffl) [inline package.json]: The version will be updated automatically on release.

**Comment by daffl** [comment]
I removed the branch protection rule and also made you an admin for the repository.

**Comment by dekelev** [comment]
Thank you!

**Pull request merged**